install/windows.md

```diff
@@ -666,6 +666,18 @@ ipython
 ```
 %matplotlib
 ```
+Ist die Ausgabe hier
+```
+Using matplotlib backend: Qt5Agg
+
+In [2]: QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-<username>'
+```
+könnt ihr _ipython_ mit `quit` beenden, und folgende Befehle in das Terminal eintragen
+```
+echo "mkdir -p /run/user/1000" >> ~/.bashrc
```

Comment:
SHould be `/tmp/user/1000`, otherwise this will not work due to missing write permissions.